config/_ytt_lib/github.com/cloudfoundry/capi-k8s-release/templates/ccng-config.lib.yml

```diff
@@ -114,7 +114,7 @@ db: &db
 
 telemetry_log_path: "/dev/null"
 log_cache:
-  url: "TODO.TODO"
+  url: #@ "log-cache.{}".format(data.values.system_domain)
```

Comment:
just for info, you can also do this: `#@ "log-cache." + data.values.system_domain`

Reply:
We initially advised to use this syntax by cf-relint. Is there are preferred syntax?

Reply:
i typically go for simplest version in a similar manner one would pick `"str"+v` vs `fmt.Sprintf("str%s", v)` in golang. no biggie though. just wanted to let yall know an alternative form. no need to change.